app/time_tracking/work_time/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils import timezone
+from django.utils.timesince import timesince
+import time
+
+class WorkTime(models.Model):
+    start_time = models.DateTimeField(blank=False, null=False)
+    end_time   = models.DateTimeField(blank=True, null=True)
+
+    work_time  = models.PositiveIntegerField(blank=True, null=True, editable=False)
+
+    start_unix_time = models.PositiveIntegerField(blank=False, null=False, editable=False)
+    end_unix_time = models.PositiveIntegerField(blank=True, null=True, editable=False)
+
+    days_count = models.PositiveIntegerField(blank=False, null=False, editable=False)
```

Comment:
`days_count` can be a calculated field

Reply:
Correct, but I used it to group the working days together to perform the statistics calculations.
I'll see what else I can do to reach the same result without this field.